### ISSUE TITLE
Error in bert_wordpiece.py in v 0.4 (0.3 OK)

### ISSUE THREAD

**Issue by abedkhooli** [huggingface/tokenizers#137]
Code that worked in v 0.3 (ex. https://github.com/huggingface/tokenizers/blob/master/bindings/python/examples/train_bert_wordpiece.py) breaks in 0.4 with type error:
```
/opt/conda/lib/python3.6/site-packages/tokenizers/implementations/bert_wordpiece.py in train(self, files, vocab_size, min_frequency, limit_alphabet, initial_alphabet, special_tokens, show_progress, wordpieces_prefix)
     92         if isinstance(files, str):
     93             files = [files]
---> 94         self._tokenizer.train(trainer, files)

TypeError: 
```

**Comment by n1t0** [comment]
Hi @abedkhooli! Thank you for letting us know about this issue. I just released `0.4.2` to fix this.

**Comment by HuanhuanZhao08** [comment]
Hello, I have the same error with version 0.12.1

Traceback (most recent call last):
  File "/home/huanhuan/Desktop/BERT-MIMIC3-ICD9-master/create_tokenizer.py", line 25, in <module>
    tokenizer.train(
  File "/home/huanhuan/miniconda3/lib/python3.9/site-packages/tokenizers/implementations/bert_wordpiece.py", line 118, in train
    self._tokenizer.train(files, trainer=trainer)
Exception: No such file or directory (os error 2)
